### PR TITLE
[Actions] Add the backport action

### PR DESCRIPTION
**Pull request by mandel-macaque**
As part of the effort to move away from jenkins, the @monojenkins bot will be retired. This adds an action to be used that does the same job.

New action can be executed via '@vs-mobiletools-engineering-service2 backport $TARGET_BRANCH'

### CODE REVIEW

File: .github/workflows/backport-trigger.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name: Backport Trigger
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  launchBackportBuild:
+    runs-on: ubuntu-latest
+    if: github.event.issue.pull_request != '' && contains(github.event.comment.body, '@vs-mobiletools-engineering-service2 backport')
```
Comment (spouliot) [inline .github/workflows/backport-trigger.yml:10]: @cadsit can't we use a shorter handle ?

Comment (cadsit) [reply .github/workflows/backport-trigger.yml:10]: @spouliot Yes -- you can replace `@vs-mobiletools-engineering-service2` on line 10 with any text you'd like to trigger a build

Comment (spouliot) [reply .github/workflows/backport-trigger.yml:10]: @mandel-macaque my choices, in preference order, are:
* @mandel-macaque
* @manuel
* @manual
* @sudo

Comment (cadsit) [reply .github/workflows/backport-trigger.yml:10]: +1 for @Manual

Comment (dalexsoto) [reply .github/workflows/backport-trigger.yml:10]: Yep let's rename it! @vs-mobiletools-engineering-service2 is jus tooooooooo long, looks like `@sudo` is free heh

Comment (mandel-macaque) [reply .github/workflows/backport-trigger.yml:10]: @spouliot @dalexsoto shall we call it @manual-macaque? Now serious, if any text is ok, I wonder if we can do /azp and use that command for everything

Comment (mandel-macaque) [reply .github/workflows/backport-trigger.yml:10]: Maybe, @azp, since with the / it will trigger the other bot :(

Comment (mandel-macaque) [reply .github/workflows/backport-trigger.yml:10]: To me honest, Manuel is perfectly valid :)

**Review by tj-devel709 — APPROVED** [review]

**Review by dalexsoto — APPROVED** [review]

**Comment by xamarin-release-manager** [comment]
❌ [Build was (probably) aborted](https://jenkins.internalx.com/job/macios/job/PR-10258/1/display/redirect)

🔥 [Jenkins job](https://jenkins.internalx.com/job/macios/job/PR-10258/1/display/redirect) (on internal Jenkins) failed in stage(s) 'Checkout' 🔥 : hudson.AbortException: script returned exit code 128

🔥 [Provisioning failed](https://jenkins.internalx.com/job/macios/job/main/605//console) 🔥
🔥 [Failed to publish results](https://jenkins.internalx.com/job/macios/job/main/605//console) 🔥

**Pull request merged**